src/main/java/io/deephaven/benchmark/generator/ColumnDefs.java

```diff
@@ -40,7 +40,7 @@ public int getCount() {
      * @return true if incremental/fixed, otherwise false.
      */
     public boolean isFixed() {
-        return isFixed;
+        return "incremental".equals(defaultDistribution);
```

Comment:
Why is "incremental" the same as "fixed"?

Reply:
"isFixed" was a flag before I added the distribution stuff.  In really mean "isIncremental()" but there where tests that used isFixed().  I decided make it backward compatible for this ticket.  Now that I look at usage, a refactor now to isIncremental() wouldn't be that bad.

Reply:
OK. Are you changing it in this PR, or merging as-is?

Reply:
I'll can change it in this PR after the meeting.

Reply:
Upon further review, it looks like isFixed() or fixed() are used in a lot of places, mostly for integration testing.  So I think it makes sense to do that in another ticket.